ZelBack/src/services/fluxNetworkHelper.js

```diff
@@ -304,7 +313,7 @@ async function checkAppAvailability(req, res) {
         res.json(dataResponse);
         return;
       }
-      const successResponse = messageHelper.createSuccessMessage(`Flux App ${appname} is available.`);
+      const successResponse = messageHelper.createSuccessMessage(`Flux Application on ${ip}:${ipPort} are available.`);
```

Comment:
@TheTrunk I think that this message refers to multiple apps?
```suggestion
      const successResponse = messageHelper.createSuccessMessage(`Flux Applications on ${ip}:${ipPort} are available.`);
```